src/components/Board/BoardContainer.tsx

```
@@ -1,38 +1,43 @@
+    { title: "To do", tasks: [] },
+    { title: "In Progress", tasks: [] },
+    { title: "Done", tasks: [] }
+];
+
+const getTaskStatus = (boardTitle: string): TaskStatus => {
+    switch (boardTitle) {
+        case "In Progress":
+            return "in-progress";
+        case "Done":
+            return "done";
+        case "To do":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)